### PR TITLE
enable svirt consoles for non-JeOS/s390 usage of svirt BACKEND

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: lib/susedistribution.pm [hunk 1]
```diff
@@ -216,7 +219,7 @@ sub init_consoles {
     }
 
     # JeOS via svirt backend
-    if (get_var('JEOS_SVIRT')) {
+    if (get_var('JEOS_SVIRT') or get_var('GENERAL_SVIRT')) {
```
Comment (coolo) [inline lib/susedistribution.pm]: what exactly makes your use case 'general'?

**Comment by coolo** [comment]
We need to differentiate between svirt usage with head or without head. GENERAL_SVIRT just doesn't express this

**Comment by mimi1vx** [comment]
I now need proprerly consoles with same wayu as JeOS without JeOS flavour ..

now It works only on s390_svirt and JeOS. Installation test hangs with `no console defined` :(

**Comment by sysrich** [comment]
I agree with @coolo , and want to avoid a pile of $FOO_SVIRT variables if we can get away with it

@mnowaksuse , is JEOS_SVIRT sufficient to be used as 'SVIRT with head'? or am I misunderstanding something here? Shouldn't JeOS's use of SVIRT be the same as 'standard SVIRT' with the exception of JeOS starts with a HDD image and what @mimi1vx is attempting to do will include an install?

**Comment by mnowaksuse** [comment]
JeOS under SVIRT and installation from DVD under SVIRT are the same here - we need to see everything from Grub on. That's one use case. The other is when we connect to YaST provided VNC server (be in under SVIRT or not) from net install. So far it seems to me that we were just abusing JEOS (and !JEOS var). JeOS is to me just a (special) case of (imaginary) "HEAD_SVIRT".

**Comment by sysrich** [comment]
LGTM

**Pull request merged**